### ISSUE TITLE
can't connect the wsl(1.0) using Warp SSh

### ISSUE THREAD

**Issue by yqmQ** [warpdotdev/Warp#2065]
### Local Shell and Version

5.8

### Remote Shell and Version

5.8.1

### Operating System and Version

MACOS 12.3

### Warp Version

v0.2022.11.01.08.03.stable_00

### I have tried commenting out my system & user rcfiles and warp still will not bootstrap

- [X] Yes

### Screenshots

When I using Warp SSH Wrapper connect  WSL```(1.0  Ubuntu 22.04.1 LTS (GNU/Linux 4.4.0-19041-Microsoft x86_64)``` ,it shows blew

<img width="1017" alt="2022-11-08 14 58 42" src="https://user-images.githubusercontent.com/99985923/200496642-6b98cc05-1e6f-48f0-bdf2-94d718bfd2ab.png">

However,When I using terminal,I can connect WSL successfully and I find some ```warp*``` files on the destop like this:
<img width="149" alt="2022-11-08 15 05 47" src="https://user-images.githubusercontent.com/99985923/200497233-e7928ce4-1c7b-45bd-abfc-b7fdceb4d324.png">

I don't know why and  hope Warp can fix the bug soon.Thanks

### Include shell xtrace output

+/etc/zsh/zshenv:15> [[ -z /home/ye/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin || /home/ye/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin == /bin:/usr/bin ]]
+warptmp.jxvkng/.zshenv:1> unsetopt ZLE
+warptmp.jxvkng/.zshenv:1> unset RCS
+warptmp.jxvkng/.zshenv:1> unset GLOBAL_RCS
+warptmp.jxvkng/.zshenv:1> printf '\x1b\x50\x24\x647b22686f6f6b223a2022496e69745368656c6c222c202276616c7565223a207b227368656c6c223a20227a7368227d7d\x9c'

**Comment by dannyneira** [comment]
Hi @yqmQ So sorry about this. Please try and disable the SSH Wrapper in the `Settings > Features`. You'll lose the blocks/completion but it should allow you to connect to WSL machine.

My guess is that you normally have a [custom prompt, theme, or plugin on the remote machine ( WSL ) and Warp is unable to parse it.](https://docs.warp.dev/features/prompt#custom-prompt-compatibility-table)

You can disable parts of your dotfiles just for Warp by using this conditional statement:

```sh

if [[ $TERM_PROGRAM != "WarpTerminal" ]]; then
##### WHAT YOU WANT TO DISABLE FOR WARP - BELOW

# Whatever you want to disable

##### WHAT YOU WANT TO DISABLE FOR WARP - ABOVE
fi

```

We have a list of incompatible tools here: <https://docs.warp.dev/help/known-issues#list-of-incompatible-tools>
And more info on custom prompts here: <https://docs.warp.dev/features/prompt>

Let me know if that helps.

**Comment by igorwojda** [comment]
I had a similar issue when establishing ssh connection - disabling  SSH Wrapper helped.

**Comment by yqmQ** [comment]
> 

yep,disable ```SSH Wrapper``` can solve the problem,but in that way,I can't use various features that warp provide.
